openai/openai.go

```diff
@@ -63,6 +63,7 @@ type ChatCompletionRequest struct {
 	FrequencyPenalty *float64        `json:"frequency_penalty"`
 	PresencePenalty  *float64        `json:"presence_penalty_penalty"`
 	TopP             *float64        `json:"top_p"`
+	MinP             *float64        `json:"min_p"`
```

Comment:
Let's omit `min_p` the OpenAI API for now: there's a few parameters that are not in the OpenAI API that should be added at some point (`num_ctx`) etc – but I don't want to break the API if we choose to do this differently later (e.g. nest these non-standard options under `options`)

Reply:
Can you share your concerns about the presence of extra parameters?
I have heard you say this several times, yet I can’t see how this can break OpenAI-compatible clients: ones that use strict adherence to OpenAI schema would just not send it ever, and therefore won’t break. Am I missing something?..